neps/utils/run_args_from_yaml.py

```diff
@@ -256,6 +256,34 @@ def process_config_key(settings: Dict, special_configs: Dict, keys: List) -> Non
                     )
 
 
+def process_pipeline_space(key, special_configs, settings):
+    if special_configs.get(key) is not None:
+        pipeline_space = special_configs[key]
```

Comment:
Never seen this combination before funnily enough, here are someother for you info:

```python
if key in special_configs:
    pipeline_space = special_configs[key]
    
## OR

pipeline_space = special_configs.get(key)
if pipeline_space is not None:
    ... # Use pipeline_space as you would
    
# OR

if (pipeline_space := special_configs.get(key)) is not None:
   # Use pipeline space as you would
```

This function is likely going to run only once and performance doesn't matter, choose your favorite :)

Reply:
yeah its like this: 

```python
pipeline_space = special_configs.get(key)
if pipeline_space is not None:
```
The key should be in the dict by default with the value None thats why the first solution would not work. I could simplify it to:

```python
if special_configs[key] is not None:
pipeline_space = special_configs[key]
```

Reply:
with .get I get for sure a None as a return, if special_configs not contains this key